javav2/usecases/monitor_dynamodb/README.md

```diff
@@ -0,0 +1,640 @@
+| Description     | This tutorial shows users how to configure an application's use of DynamoDB to monitor performance by using the AWS SDK for Java (v2). |
+| Audience        | Developer (beginner)                                                                                                                   |
+| Required skills | Java, Maven                                                                                                                            |
+
+## Purpose
+
+In an application that uses DynamoDB, it can be important to understand and monitor DynamoDB’s contributions to
+the application's performance. This tutorial discusses the following ways to monitor performance.
+* Configure client-side performance metrics.
+* View metrics from DynamoDB and from the SDK.
+* Set up alarms on those metrics.
+* Set up Cloudwatch Contributor Insights to dive into specific issues with usage patterns. 
+ 
+This tutorial does not cover general best-practices for DynamoDB. For information on best practices see [Best practices for designing
+and architecting with DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/best-practices.html).
+This tutorial uses the DynamoDB enhanced client, for more information see the [DynamoDB Enhanced Client API topic](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/dynamodb-enhanced-client.html)
+in the AWS SDK for Java 2.x Developer Guide.
+
+In this tutorial, we create a sample application that uses DynamoDB that demonstrates how to monitor DynamoDB
+performance.
+
+This tutorial uses the following AWS services.
+
+- Amazon DynamoDB
+- Amazon CloudWatch
+
+## Prerequisites
+
+To complete the tutorial, you need the following:
+
++ Access to AWS services
++ A Java IDE (this tutorial uses IntelliJ version 2023.3.6)
++ Java version 8 (or higher) SDK
+
+> [!IMPORTANT]
+> + The AWS services used in this tutorial are included in the [AWS Free Tier](https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc).
+> + This code has not been tested in all AWS Regions. Some AWS services are available only in specific Regions. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-greeting-services).
+> + Running this code might result in charges to your AWS account.
+> + Be sure to delete all the resources that you create while going through this tutorial so that you won't be charged.
+
+### Set up your development environment
+Make sure that you have properly set up your development environment. For information, see
+[Set up the AWS SDK for Java 2.x](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html).
+
+## Create a sample application
+### Create an IntelliJ project
+As a first step, create an IntelliJ project.
+
+1. From the IntelliJ IDE menu, choose **File** -> **New** -> **Project ...**.
```

Comment:
Is the style , or Arrows between UI elements?

Reply:
I've not found guidance on how to navigate a hierarchical menu. Do you know of any? 
So, I've often seen the use of an arrow to separate the levels.

Reply:
ok